src/schema/workflow_execution_activity.yaml

```diff
@@ -23,6 +23,30 @@ subsets:
       Subset consisting of just the workflow execution activities
 
 classes:
+  WorkflowChain:
+    description: >-
+      A workflow chain is a sequence of workflow execution activities that are related to each other in some way.
```

Comment:
change sequence to collection

Reply:
Add the sequence of collection can be discover by has_input and has_output patterns